plugin.json

```diff
@@ -27,6 +27,12 @@
         "display_name": "Apply plugin to updated posts as well as new posts",
         "type": "bool",
         "default": false
+      },
+      {
+        "key": "AdminUsernames",
```

Comment:
Generally we only accept user IDs in fields like this. Since typos can happen and usernames can be changed, it is a minor security risk. 

We hope to fix the UX with a user picker component that would ease the process for admins. 

Do you mind updating this to use IDs instead of names?